pages/people.md

```diff
@@ -169,6 +169,7 @@ GitHub also has a [contributor list][github-contributors] based on commits.
 | Shivakumar Gangamath|                                                                   | [IST][IST]            |
 | Steve Loughran      | [Hortonworks][HORTONWORKS]                                        | [GMT][GMT]/[BST][BST] |
 | Supun Kamburugamuva |                                                                   |                       |
+| Swastik Pal         |                                                                   | [IST](https://www.timeanddate.com/time/zones/ist)|
```

Comment:
```suggestion
| Swastik Pal         |                                                                   | [IST][IST]            |
```

This file is processed when it is published and the timezone link will be added (it exists at the end of this file) so you don't need the markdown syntax here.

Reply:
[IST] is resolved to Irish Summer Time (https://www.timeanddate.com/time/zones/ist-ireland), not Indian Standard Time (https://www.timeanddate.com/time/zones/ist).

Reply:
There is a precedence [here](https://github.com/kitswas/accumulo-website/blob/7746e11f63c010bbf81ffe97a2bd422b7d0b39f0/pages/people.md?plain=1#L130).

Refer to https://github.com/apache/accumulo-website/pull/288 for details.
@ctubbsii Needs fixing https://github.com/apache/accumulo-website/pull/288#discussion_r670731431.

Reply:
I submitted https://github.com/apache/accumulo-website/pull/320 to update the definitions and adds you as a contributor.